### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by pmartinolli**
Translation of the missing items. 

Also, in this submitted translation, I proposed to rename "backlinks" into "rétroliens" instead of "liens retour" (as I did this summer), the reason is : the translation is still correct, its shorter and also its much more easy to pronounce. I experienced it when I was giving workshops on Obsidian to my students.

### CODE REVIEW

**Comment by Write** [comment]
Hello @pmartinolli,

A PR for a french translation is open since a few days here : https://github.com/obsidianmd/obsidian-translations/pull/173

With Olive#3715 and Alain Leroy#7197 (on discord) we discussed about keeping the litteral word 'tag' instead of changing it to its french translation because it seems to be a well known word in french.

You translated 'Front Matter' (https://jekyllrb.com/docs/front-matter/) with a translation 'Introduction' but i think we should keep the original name aka Front Matter (Same goes for Publish plugin name, altough it's seems more ambiguous).

Apart from that, you fixed a lot of translations and we definitely should merge them with #173 but i'm not sure how.

___

Bonjour @pmartinolli, 

Un PR pour une traduction fr est ouverte depuis quelques jours : https://github.com/obsidianmd/obsidian-translations/pull/173

Par mail avec Olive#3715 et Alain Leroy#7197 (sur discord) nous avions décidé de garder le mot 'tag' plutôt que de le changer par balise (ou autre) car il nous semblait que le terme était déjà très utilisé et compris en français.

Vous avez traduit 'Front Matter' (https://jekyllrb.com/docs/front-matter/) par 'Introduction' je ne pense pas que cela soit approprié de changer le nom propre, cela va aussi pour le nom du plugin 'Publish', mais j'admet que c'est ambigue.

En dehors de ça les changements et corrections m'ont l'air super mais il y aura un conflit avec le #173, je ne vois pas comment fusionner les deux pour le moment.

**Comment by ericaxu** [comment]
Hi all, I merged in #173 earlier and it seems that has caused some conflicts.

Any suggest on which version we should adopt?

**Comment by Write** [comment]
> 
> 
> Hi all, I merged in #173 earlier and it seems that has caused some conflicts.
> 
> Any suggest on which version we should adopt?

I sent a mail to Olive#3715 to Alain Leroy#7197 to get them to answer here. 

From my POV, keeping #173 seems the more logical solution to get a proper merge. I think @pmartinolli should change the PR accordingly.

File: fr.json [hunk 1]
```diff
@@ -14,24 +14,24 @@
 			"option-readable-line-description": "Limite la longueur de ligne maximale. Affiche moins de contenu à l'écran, mais rend les longs paragraphes plus lisibles.",
 			"option-strict-line-break": "Sauts de ligne stricts",
 			"option-strict-line-break-description": "En mode prévisualisation, ignore les sauts de ligne selon les spécifications Markdown. Désactivez cette option pour que les sauts de ligne uniques s'affichent en mode prévisualisation.",
-			"option-show-frontmatter": "Afficher frontmatter",
-			"option-show-frontmatter-description": "En mode prévisualisation, affiche frontmatter en bloc de code. Désactiver pour cacher la section frontmatter.",
-			"option-use-markdown-links": "Utiliser les liens markdown",
-			"option-use-markdown-links-description": "La suggestion de lien complète automatiquement les liens et images markdown au lieu des wikilinks.",
+			"option-show-frontmatter": "Afficher l'introduction",
```
Comment (Write) [inline fr.json:17]: I'm not okay with this translation, comment in Conversation tab.

File: fr.json [hunk 2]
```diff
@@ -14,24 +14,24 @@
...
+			"option-show-frontmatter-description": "En mode prévisualisation, affiche l'introduction en bloc de code. Désactiver pour cacher la section de l'introduction.",
```
Comment (Write) [inline fr.json:18]: I'm not okay with this translation, comment in Conversation tab.

File: fr.json [hunk 3]
```diff
@@ -14,24 +14,24 @@
...
+			"option-use-markdown-links": "Utiliser les liens Markdown",
```
Comment (Write) [inline fr.json:19]: Ok

File: fr.json [hunk 4]
```diff
@@ -14,24 +14,24 @@
...
+			"option-use-markdown-links-description": "La suggestion de lien complète automatiquement les liens et les images Markdown au lieu des wikilinks.",
```
Comment (Write) [inline fr.json:20]: Ok

File: fr.json [hunk 5]
```diff
@@ -14,24 +14,24 @@
...
 			"option-auto-pair-brackets": "Association automatique des crochets",
 			"option-auto-pair-brackets-description": "Associer automatiquement les crochets et les guillemets.",
 			"option-auto-pair-markdown": "Association automatique des syntaxes Markdown",
 			"option-auto-pair-markdown-description": "Associer automatiquement les symboles des caractères gras et italiques.",
 			"option-smart-indent-lists": "Indentation intelligente des listes",
 			"option-smart-indent-lists-description": "Prise en charge de l'indentation des marqueurs de liste pour vous.",
-			"option-fold-heading": "Pliage l'en-tête",
+			"option-fold-heading": "Pliage de l'en-tête",
```
Comment (Write) [inline fr.json:27]: Ok

File: fr.json [hunk 6]
```diff
@@ -14,24 +14,24 @@
...
 			"option-fold-heading-description": "Permet de plier toute la section sous un en-tête.",
 			"option-fold-indent": "Pliage de l'indentation",
-			"option-fold-indent-description": "Autoriser le pliage des portions de même indentation, tel que les listes.",
-			"option-show-line-number": "Afficher le numéro de ligne",
-			"option-show-line-number-description": "Afficher le numéro de ligne dans la marge.",
+			"option-fold-indent-description": "Autorise le pliage des portions de même indentation, tels que les listes.",
```
Comment (Write) [inline fr.json:30]: Ok

File: fr.json [hunk 7]
```diff
@@ -14,24 +14,24 @@
...
+			"option-show-line-number": "Afficher numérotation de ligne",
```
Comment (Write) [inline fr.json:31]: Maybe "Afficher la numérotation de ligne" ?

File: fr.json [hunk 8]
```diff
@@ -14,24 +14,24 @@
...
+			"option-show-line-number-description": "Affiche la numérotation de ligne dans la marge.",
```
Comment (Write) [inline fr.json:32]: Ok

File: fr.json [hunk 9]
```diff
@@ -14,24 +14,24 @@
...
 			"option-use-tabs": "Utiliser des tabulations",
-			"option-use-tabs-description": "Utiliser des tabulations pour indenter lorsque vous appuyez sur la touche \"Tab\". Désactivez cette option pour utiliser des espaces.",
+			"option-use-tabs-description": "Utilise des tabulations pour indenter lorsque vous appuyez sur la touche \"Tab\". Désactivez cette option pour utiliser des espaces.",
```
Comment (Write) [inline fr.json:34]: Ok

File: fr.json [hunk 10]
```diff
@@ -54,7 +54,7 @@
 			"option-choice-current-folder": "Même dossier que le fichier",
 			"option-choice-specified-folder": "Dans le dossier spécifié ci-dessous",
 			"option-new-file-folder-path": "Dossier dans lequel créer de nouvelles notes",
-			"option-new-file-folder-path-description": "Les notes nouvellement créées apparaîtront sous ce dossier. Les paramètres des modules d'extension primeront en cas de conflit.",
+			"option-new-file-folder-path-description": "Les notes nouvellement créées apparaîtront dans ce dossier. Les paramètres des modules d'extension primeront en cas de conflit.",
```
Comment (Write) [inline fr.json:57]: Ok

File: fr.json [hunk 11]
```diff
@@ -66,12 +66,12 @@
 		"appearance": {
 			"name": "Apparence",
 			"option-base-mode": "Mode de base",
-			"option-base-mode-description": "La palette de couleurs de base d'Obsidian.",
+			"option-base-mode-description": "La palette des couleurs de base d'Obsidian.",
```
Comment (Write) [inline fr.json:69]: Ok

File: fr.json [hunk 12]
```diff
@@ -66,12 +66,12 @@
...
 			"dark-mode": "Mode sombre",
 			"light-mode": "Mode clair",
-			"option-community-themes": "Ouvrir les thèmes communautaires",
-			"option-community-themes-description": "Prévisualiser et utiliser des thèmes incroyables créés par la communauté.",
-			"see-community-themes": "Voir les thèmes communautaires"
+			"option-community-themes": "Ouvrir les thèmes de la communauté",
```
Comment (Write) [inline fr.json:72]: Ok

File: fr.json [hunk 13]
```diff
@@ -66,12 +66,12 @@
...
+			"option-community-themes-description": "Prévisualiser et utiliser des thèmes incroyables créés par la communauté Obsidian.",
+			"see-community-themes": "Voir les thèmes de la communauté"
```
Comment (Write) [inline fr.json:74]: Ok

File: fr.json [hunk 14]
```diff
@@ -98,7 +98,7 @@
 			"label-new-version-ready": "Une nouvelle version est prête pour l'installation. ",
 			"label-relaunch-to-update": "Relancer l'application pour faire la mise à jour.",
 			"label-disabled-updates": "Les mises à jour sont désactivées.",
-			"label-checking-for-updates": "Recherche de mises à jour...",
+			"label-checking-for-updates": "Recherche des mises à jour...",
```
Comment (Write) [inline fr.json:101]: Ok

File: fr.json [hunk 15]
```diff
@@ -124,11 +124,11 @@
 			"name": "Compte",
 			"label-welcome": "Bienvenue, {{name}}!",
 			"label-your-account-email": "Votre compte est ",
-			"label-insider": "insider",
-			"label-supporter": "supporter",
+			"label-insider": "utilisateur en accès anticipé",
```
Comment (Write) [inline fr.json:127]: Ok

File: fr.json [hunk 16]
```diff
@@ -124,11 +124,11 @@
...
+			"label-supporter": "supporteur",
 			"label-vip": "VIP",
 			"label-role-text": "Vous êtes un {{- role}}!",
-			"label-insider-build-access": "Merci de votre soutien. Vous avez accès à nos pré-versions internes.",
+			"label-insider-build-access": "Merci de votre soutien. Vous avez accès à nos pré-versions en accès anticipé.",
```
Comment (Write) [inline fr.json:131]: Ok

File: fr.json [hunk 17]
```diff
@@ -185,7 +185,7 @@
 		"msg-switched-to-preview": "Changement du mode d'affichage par défaut à prévisualisation.",
 		"msg-switched-to-edit": "Changement du mode d'affichage par défaut à éditeur.",
 		"msg-upgrade-installer": "Pour utiliser cette fonctionnalité, veuillez réinstaller Obsdian avec le dernier programme d'installation disponible sur notre site web.",
-		"tooltip-restore-default-settings": "Restore default settings",
+		"tooltip-restore-default-settings": "Restoration des paramètres par défaut",
```
Comment (Write) [inline fr.json:188]: Ok

File: fr.json [hunk 18]
```diff
@@ -294,26 +294,26 @@
 		"button-delete-do-not-ask-again": "Supprimer et ne plus redemander",
 		"label-existing-backlink": "Il y a actuellement {{links}} pointant vers cette note.",
 		"label-delete-file": "Supprimer le fichier",
-		"button-cancel": "Cancel"
+		"button-cancel": "Annuler"
```
Comment (Write) [inline fr.json:297]: Ok

File: fr.json [hunk 19]
```diff
@@ -294,26 +294,26 @@
...
 	},
 	"plugins": {
 		"name": "Modules d'extension",
 		"file-explorer": {
 			"name": "Explorateur de fichiers",
 			"desc": "Voir tous les fichiers de votre projet.",
 			"action-create-note": "Créer une nouvelle note",
-			"action-create-note-in-new-pane": "Create note in new pane",
+			"action-create-note-in-new-pane": "Créer une nouvelle note dans un nouveau volet",
```
Comment (Write) [inline fr.json:305]: Ok

File: fr.json [hunk 20]
```diff
@@ -294,26 +294,26 @@
...
+			"action-reveal-active-file": "Afficher le fichier actif dans l'explorateur de fichiers",
 			"label-sort-a-to-z": "Trier par nom de fichier (A à Z)",
 			"label-sort-z-to-a": "Trier par nom de fichier (Z à A)",
-			"label-sort-new-to-old": "Trier par heure d'édition (nouveau à ancien)",
-			"label-sort-old-to-new": "Trier par heure d'édition (ancien à nouveau)",
-			"label-sort-created-old-to-new": "Created time (old to new)",
-			"label-sort-created-new-to-old": "Created time (new to old)",
+			"label-sort-new-to-old": "Trier par date d'édition (nouveau à ancien)",
+			"label-sort-old-to-new": "Trier par date d'édition (ancien à nouveau)",
+			"label-sort-created-old-to-new": "Trier par date de création (ancien à nouveau)",
```
Comment (Write) [inline fr.json:315]: Already translated in #173

File: fr.json [hunk 21]
```diff
@@ -294,26 +294,26 @@
...
+			"label-sort-created-new-to-old": "Trier par date de création (nouveau à ancien)",
```
Comment (Write) [inline fr.json:316]: Already translated in #173

File: fr.json [hunk 22]
```diff
@@ -294,26 +294,26 @@
...
-			"action-reveal-active-file": "Affiche le fichier actif dans l'explorateur de fichiers",
+			"action-reveal-file": "Afficher le fichier dans l'explorateur de fichiers",
+			"action-reveal-active-file": "Afficher le fichier actif dans l'explorateur de fichiers",
 			"label-sort-a-to-z": "Trier par nom de fichier (A à Z)",
 			"label-sort-z-to-a": "Trier par nom de fichier (Z à A)",
-			"label-sort-new-to-old": "Trier par heure d'édition (nouveau à ancien)",
-			"label-sort-old-to-new": "Trier par heure d'édition (ancien à nouveau)",
-			"label-sort-created-old-to-new": "Created time (old to new)",
-			"label-sort-created-new-to-old": "Created time (new to old)",
+			"label-sort-new-to-old": "Trier par date d'édition (nouveau à ancien)",
```
Comment (Write) [inline fr.json:313]: Ok

File: fr.json [hunk 23]
```diff
@@ -294,26 +294,26 @@
...
+			"label-sort-old-to-new": "Trier par date d'édition (ancien à nouveau)",
```
Comment (Write) [inline fr.json:314]: Ok

File: fr.json [hunk 24]
```diff
@@ -294,26 +294,26 @@
...
 			"desc": "Voir tous les fichiers de votre projet.",
 			"action-create-note": "Créer une nouvelle note",
-			"action-create-note-in-new-pane": "Create note in new pane",
+			"action-create-note-in-new-pane": "Créer une nouvelle note dans un nouveau volet",
 			"action-new-note": "Nouvelle note",
 			"action-new-folder": "Nouveau dossier",
 			"action-change-sort": "Modifier l'ordre de tri",
-			"action-reveal-file": "Affiche le fichier dans l'explorateur de fichiers",
-			"action-reveal-active-file": "Affiche le fichier actif dans l'explorateur de fichiers",
+			"action-reveal-file": "Afficher le fichier dans l'explorateur de fichiers",
```
Comment (Write) [inline fr.json:309]: Ok

File: fr.json [hunk 25]
```diff
@@ -334,33 +334,33 @@
 			"label-more-context": "Afficher plus de contexte",
 			"prompt-start-search": "Saisir pour lancer la recherche...",
 			"label-match-text": "Correspond au texte : ",
-			"label-match-regex": "Correspond au regex: ",
-			"label-match-exact-text": "Contient le texte exact : ",
-			"label-match-all": "Contient tous ces termes: ",
-			"label-match-any": "Contient au moins l'un de ces termes: ",
-			"label-excluding": "Exclure: ",
+			"label-match-regex": "Correspond au regex : ",
+			"label-match-exact-text": "Contient l'expression exacte : ",
```
Comment (Write) [inline fr.json:338]: Ok

File: fr.json [hunk 26]
```diff
@@ -334,33 +334,33 @@
 			"label-more-context": "Afficher plus de contexte",
 			"prompt-start-search": "Saisir pour lancer la recherche...",
 			"label-match-text": "Correspond au texte : ",
-			"label-match-regex": "Correspond au regex: ",
-			"label-match-exact-text": "Contient le texte exact : ",
-			"label-match-all": "Contient tous ces termes: ",
-			"label-match-any": "Contient au moins l'un de ces termes: ",
-			"label-excluding": "Exclure: ",
+			"label-match-regex": "Correspond au regex : ",
```
Comment (Write) [inline fr.json:337]: Ok

File: fr.json [hunk 27]
```diff
@@ -334,33 +334,33 @@
...
+			"label-match-all": "Contient tous ces termes : ",
+			"label-match-any": "Contient au moins l'un de ces termes : ",
+			"label-excluding": "Exclure : ",
 			"label-case-sensitive": "Sensible à la casse",
 			"label-case-insensitive": "Insensible à la casse",
 			"label-match-file-path": "Correspond au chemin d'accès au fichier : ",
 			"label-match-file-name": "Correspond au nom du fichier : ",
 			"label-match-content": "Correspond au contenu : ",
-			"label-match-tag": "Match tag: ",
+			"label-match-tag": "Correspond au hashtag : ",
```
Comment (Write) [inline fr.json:347]: C'est pour matcher un 'tag' bien qu'il contient effectivement un hashtag, je serais d'avis de garder le mot tag tout simplement. Proposition : "Correspond au tag : "

File: fr.json [hunk 28]
```diff
@@ -369,67 +369,67 @@
...
+			"option-repel-force": "Force de repoussement",
+			"tooltip-open-graph-settings": "Ouvrir les propriétés graphiques",
+			"msg-screenshot-copied": "Capture d'écran copiée dans le presse-papier."
 		},
 		"backlinks": {
-			"name": "Liens retours",
-			"desc": "Afficher le nombre de liens retour dans la barre d'état.",
-			"action-open": "Ouvrir les liens retour",
-			"action-open-for-current": "Ouvrir les liens retour du fichier courant",
+			"name": "Rétroliens",
```
Comment (Write) [inline fr.json:406]: For all backlinks to Rétroliens : Ok

File: fr.json [hunk 29]
```diff
@@ -694,7 +694,7 @@
 		"link-with-count_plural": "{{count}} liens",
 		"file-with-count": "{{count}} fichier",
 		"file-with-count_plural": "{{count}} fichiers",
-		"backlink-with-count": "{{count}} lien retour",
-		"backlink-with-count_plural": "{{count}} liens retour"
+		"backlink-with-count": "{{count}} rétrolien",
```
Comment (Write) [inline fr.json:697]: Ok

File: fr.json [hunk 30]
```diff
@@ -439,7 +439,7 @@
 		},
 		"custom-css": {
 			"name": "CSS personnalisé",
-			"desc": "Lit \"obsidian.css\" dans le projet et l'applique.",
+			"desc": "Lire \"obsidian.css\" dans le projet et l'appliquer.",
```
Comment (Write) [inline fr.json:442]: Ok

File: fr.json [hunk 31]
```diff
@@ -334,33 +334,33 @@
...
 			"instruction-create": "pour créer",
 			"instruction-dismiss": "pour annuler",
-			"label-no-note-create-new": "Aucune note trouvée. Appuyer sur Entrer pour en créer une nouvelle.",
-			"prompt-type-file-name": "Saisir le fichier vers lequel basculer ou créer...",
-			"label-enter-to-create": "Entrer pour créer"
+			"label-no-note-create-new": "Aucune note trouvée. Appuyer sur Entrée pour en créer une nouvelle.",
+			"prompt-type-file-name": "Saisissez le fichier vers lequel basculer ou créer...",
+			"label-enter-to-create": "Entrée pour créer"
 		},
 		"graph-view": {
```
Comment (Write) [inline fr.json:365]: This whole section has been translated in #173

File: fr.json [hunk 32]
```diff
@@ -592,97 +592,97 @@
 			"desc": "Activez l'effet de translucidité pour améliorer la sensation de profondeur. À utiliser de préférence avec le mode sombre. Non pris en charge sous Linux."
 		},
 		"publish": {
```
Comment (Write) [inline fr.json:594]: This whole section has been translated in #173

**Comment by pmartinolli** [comment]
I am OK with all comments of Joffrey. Thank you.

**Comment by ericaxu** [comment]
Thank you, I'll wait for this PR to stabilize. Let me know if you'd like me to resolve conflicts by using master (i.e. the changes from #173).

**Comment by Write** [comment]
> 
> 
> I am new to PR : is it better for example that I close my PR, wait you validate the previous one (#173 I guess) and then redo a translation of som missing items (if needed) ? Thank you.

Hi,
Thanks for your answer.
#173 has been merged, i think you can close your PR and re-open if it's more convenient yes

Have a great day,
Jo

**Comment by pmartinolli** [comment]
I am closing this PR. Thanks.

**Comment by ericaxu** [comment]
Thank you, sorry for the trouble @pmartinolli . I should have merged #173 earlier.

**Pull request closed without merging**